apps/user-ms/src/main/java/it/pagopa/selfcare/user/service/UserNotificationServiceImpl.java

```diff
@@ -166,6 +167,7 @@ private static String retrieveMail(UserResource user, UserInstitution institutio
         } else {
             email = certEmail.getEmail().getValue();
         }
+        log.info("retrieved Mail for user");
```

Comment:
you could add userId, without reference could not be useful as information.....maybe in debug mode